crates/sui-config/src/node.rs

```diff
@@ -233,11 +233,17 @@ pub struct ExecutionTimeObserverConfig {
     /// If unspecified, this will default to `50_000`.
     pub object_utilization_cache_size: Option<NonZeroUsize>,
 
-    /// If true, the execution time observer will report per-object utilization metrics.
-    /// Warning: This metric may have very large cardinality.
+    /// If true, the execution time observer will report per-object utilization metrics
+    /// with full object IDs. When set, the metric can have a high cardinality, so this
+    /// should not be used except in controlled tests where there are a small number of
+    /// objects.
+    ///
+    /// If false, object utilization is reported using hash(object_id) % 16 as the key,
```

Comment:
we can probably safely go much higher than 16 - any reason not to?

Reply:
I'm not sure there's a safe/unsafe limit, isn't it just about how much load we add to the collectors?

Reply:
I think it'd be useful to set this to a higher value, a 32 or 64 cardinality metric is still very minimal load IIUC and then we'd have a bit more visibility when the network grows. Up to you though